2023-07_Pilot-ER/sections/04-03-dri.adoc

```diff
@@ -4,33 +4,40 @@
+
+With close to 500 data, AI, machine learning and pricing experts, the Intact Data Lab has deployed almost 300 AI models in production to-date, helping transform outperformance advantage. It is focused on improving risk selection and making  operations as efficient as possible while creating outstanding interactions 
+with customers. Within Intact’s Data Lab, the Centre for Climate and Geospatial Analytics (CCGA) uses weather, climate, and geospatial data along with machine learning models to develop risk maps and other specialized products to the business.
+
+==== Scope of Work
+The project centered the efforts around these challenges:
+
+ * Identify current usages of wildfire risk maps at Intact by interviewing various business units;
+ * Revisit and update previous wildfire hazard map, using external open data sources;
+ * Identify and seek collaboration opportunities with pilot participants;
+ * Inform internal architectural, infrastructure and procurement processes of new geospatial standards and trends;
+ * Identify and develop insurance wildfires risk use cases, to build Disaster and Climate Resilience.
+
+These activities should, when possible, align with the best practices and standards of the OGC, as well as current and proposed themes in OGC's DWG.
+
+==== Approach
+To develop its component, Intact migrated its previous proprietary wildfire hazard model to a new private on-premise data science environment. For all key inputs to the model, external connections to several open data repositories were established. To facilitate these access tests, early in the project, several public open source - climate model outputs, Earth observation, weather, geospatial - datasets were vetted by the appropriate cybersecurity boards. The tests informed experts of changes in platforms offerings, of new data products specifications and of current authoritative scientific references. The following datasets were accessed specifically to produce an updated wildfire hazard map (GeoTiff), and to compute regional risk indicator by postal codes (CSV):
 
-- Inputs:
   * National fire database polygon data
   * Fire Weather Index (FWI) daily maps
   * Land cover maps
   * Drought conditions
   * Digital Elevation Model (DEM)
   * Population density
 
-- Outputs:
-  * Wildfire hazard map, in GeoTiff
-  * Regional risk indicators, as tabular output in CSV format
+In parallel, CCGA polled IFC's stakeholders about current and potential uses of wildfire risk products within their operations. This information was used to identify use cases supporting this pilot project, as well as prospective proof-of-concepts for wildfire resiliency. Additionally, the team adopted a multi-hazard approach, seeking to apply learnings obtained from flood risk mapping, benchmarking and decision making. For instance, CCGA routinely manipulate exposure data representing insured risks throughout the country, as well as claims received for previous catastrophic events.
```

Comment:
The term "multi-hazard" seems to suggest that we're developing a combined wildfire/flood model.

Reply:
Adjusted wording